.travis.yml

```diff
@@ -26,28 +26,35 @@ before_install:
   - nvm use 8
   - npm install -g yarn
 
-script:
-  - make validate-commit
-  - make lint
-  - make test-frontend
-  - make test-coverage
-
-before_deploy:
-  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
-  - sudo apt-get update
-  - sudo apt-get -y install docker-ce
-  - make packages
-
-deploy:
-  provider: releases
-  api_key: $GITHUB_TOKEN
-  file_glob: true
-  file: build/*.tar.gz
-  skip_cleanup: true
-  on:
-    tags: true
-    go: 1.11.x
-
-after_deploy:
-  - make docker-push-latest
+stages:
+  - name: test
+  - name: release
+    if: tag IS present
+
+jobs:
+  include:
+    - name: 'Tests'
+      stage: test
+      script:
+        - make validate-commit
+        - make lint
+        - make test-frontend
+        - make test-coverage
+    - name: 'Release'
+      stage: release
+      script:
+        - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+        - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+        - sudo apt-get update
+        - sudo apt-get -y install docker-ce
```

Comment:
I believe we can simplify it (just remove everything from the start of `script` to this line and add `services:  - docker` as in any other our project).
I'm not sure why we have it here, most probably some legacy from very old ci.

Reply:
Do you depend on any features of the latest Docker? I have those lines in some of my travis files because the docker version Travis gives you is quite old. Just want to double check this before breaking deployment

Reply:
No, we don't. The service itself is just a simple go application with embedded js and deployment is proven to work with travis docker in v1 branch (I request the change only because the PR changes ci to a new one).